sts-mock/README.md

```diff
@@ -57,6 +57,20 @@ npm run lint
 ```
 
 ## Deploy to `dev`
+
+### Testing changes with a custom backend-api stack
+
+Update `ASYNC_BACKEND_BASE_URL` environment variable in the STS Mock [template](../template.yaml). The new value for this URL should be:
+
+```bash
+https://sessions-<backend-api-stack-name>.review-b-async.dev.account.gov.uk
+```
+
+Please note:
+
+* You can still update this value even if you have not actually deployed a `backend-api` custom stack yet
+* You can skip this step if you are deploying/testing the STS mock in isolation
```

Comment:
Is this true? The STS mock calls the backend well-known endpoint 👀

Reply:
By skip I mean skip updating the variable, by default it will call the main backend-api well known endpoint, I.e. it doesn't need to call a custom be-api to test in isolation. Or is this also not true?

Reply:
Yep it should be fine